docs/about/index.md

```diff
@@ -4,179 +4,139 @@ title: About the Design System
-    </div>
-    <div class="item">
-      <h3>Shared language and vision</h3>
-      <p>We leverage similar elements as PatternFly, so designers no longer need to reinvent the wheel when choosing components. For example, an <a href="/elements/accordion/">Accordion</a> used in a Red&nbsp;Hat web application will look the same when used on a website.</p>
-    </div>
-    <div class="item">
-      <img src="{{ '/assets/about/shared-language.svg' | url }}" alt="Shared language">
-    </div>
-    <div class="item">
-      <img src="{{ '/assets/about/web-components.svg' | url }}" alt="Web components">
-    </div>
-    <div class="item">
-      <h3>Web components</h3>
-      <p>Our team is building an open source web component-driven development system to build scalable UI elements. This <a href="https://patternflyelements.com/" target="_blank">open source community project</a> supports many components in our design system.</p>
-    </div>
-  </div>
-</section>
+<section class="intro">
+
+## We create Red Hat digital experiences
+
+Red Hat teams work across the world in a range of disciplines, from UX research and digital design to web development and content strategy. The Red Hat Design System (RHDS) for digital experiences allows designers and developers to build branded user experiences consistently and to make every digital interaction with Red Hat reflect our core brand traits: open, authentic, helpful, and brave.
 
-<hr>
-
-<section id="fundamentals" aria-labelledby="aria__heading-fundamentals">
-  <h2 id="aria__heading-fundamentals">Fundamentals</h2>
-  <div class="grid sm-two-columns">
-    <div class="item">
-      <img src="{{ '/assets/about/icon-flexible.svg' | url }}" alt="Flexibility">
-      <h3>Flexible</h3>
-      <p>Our components can be arranged in a number of ways and CSS variables can be used for further customization.</p>
-    </div>
-    <div class="item">
-      <img src="{{ '/assets/about/icon-accessible.svg' | url }}" alt="Accessibility">
-      <h3>Accessible</h3>
-      <p>The goal of our design system is to create meaningful experiences that work for everyone, regardless of ability.</p>
-    </div>
-    <div class="item">
-      <img src="{{ '/assets/about/icon-consistent.svg' | url }}" alt="Consistency">
-      <h3>Consistent</h3>
-      <p>Our documentation and tools streamline collaboration so teams can create consistent and on-brand experiences.</p>
-    </div>
-    <div class="item">
-      <img src="{{ '/assets/about/icon-scalable.svg' | url }}" alt="Scalability">
-      <h3>Scalable</h3>
-      <p>Our system enables teams to work concurrently across the Red&nbsp;Hat system of websites and beyond.</p>
-    </div>
-  </div>
-  <rh-cta variant="secondary">
-    <a href="/get-started">Get started</a>
-  </rh-cta>
 </section>
+
+## We’re guided by our core principles
+
+<div class="grid sm-two-columns">
+  <rh-card>
```

Comment:
Any idea why the icon cards look so funky here? I think there's something going on with the icon in the header slot.

Reply:
probably the aspect-ratio: 1/1

dc127f14 fixes, but these cards should probably have body content

Reply:
looks like the cards' body paragraphs weren't marked up properly.

765cb0cd should fix